arn/User.go

```diff
@@ -56,6 +57,7 @@ type User struct {
 	OS           UserOS       `json:"os" private:"true"`
 	Location     *Location    `json:"location" private:"true"`
 	FollowIDs    []UserID     `json:"follows"`
+	APIToken     uuid.UUID    `json:"apitoken" private:"true"`
```

Comment:
"apitoken" should probably be "apiToken" in JSON.

Just something to keep in mind here is that users in the future might want to register multiple "applications", hence multiple API tokens. I'm guessing you probably thought of that already and went for the simple implementation first.